xendit-android/src/main/java/com/xendit/Xendit.java

```diff
@@ -482,6 +482,7 @@ private void _createToken(Card card, String amount, boolean shouldAuthenticate,
         BaseRequest request = new BaseRequest<>(Request.Method.POST, CREATE_CREDIT_CARD_TOKEN_URL, Authentication.class, new DefaultResponseHandler<>(handler));
         request.addHeader("Authorization", basicAuthCredentials.replace("\n", ""));
         request.addHeader("x-client-identifier", CLIENT_IDENTIFIER);
+        request.addHeader("for-user-id", onBehalfOf);
```

Comment:
@IreneGohtami , turns out we don't need the logic to only send this header when the value is not empty ahaha. wanna change the ios one as well?